views/proposal/create-proposal.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/** @var string $error  */
+use yii\helpers\Html; ?>
+
+<?= 'CREATE PROPOSAL'; ?>
```

Comment:
🤔

Reply:
My bad, I change that into a beautiful h1 tag :D